### PR TITLE
Club: implement first version of post list pattern for index page

### LINKED ISSUE #6225: club: Templates: Index
Implement the template of the index page following the designs:

image

### PR DESCRIPTION
**Pull request by matiasbenedetto**
#### Changes proposed in this Pull Request:
Club: implement the first version of the post list pattern for the index page

#### Related issue(s):
https://github.com/Automattic/themes/issues/6225

### CODE REVIEW

File: club/patterns/post-list.php [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+<!-- wp:query-pagination -->
+<!-- wp:query-pagination-previous /-->
+
+<!-- wp:query-pagination-numbers /-->
+
+<!-- wp:query-pagination-next /-->
+<!-- /wp:query-pagination -->
+
+<!-- wp:query-no-results -->
+<!-- wp:paragraph {"placeholder":"Add text or blocks that will display when the query returns no results."} -->
```
Comment (jffng) [inline club/patterns/post-list.php]: Should the placeholder should be escaped + marked for translation?

**Review by jffng — COMMENTED** [review]
Figma | This PR
--- | ---
Screen Shot 2022-07-14 at 10 09 51 AM | Screen Shot 2022-07-14 at 10 10 06 AM


Some overall comments on the design:

- The post title doesn't look the correct font weight or font size (should be the equivalent of 40px).
- I also think the date and read more button text are too small compared to the design.
- The spacing between the top and bottom separators should match

Also, what should we do when there is no featured image? Currently it looks like:

Screen Shot 2022-07-14 at 10 13 16 AM

**Comment by matiasbenedetto** [comment]
> The post title doesn't look the correct font weight or font size (should be the equivalent of 40px).
I also think the date and read more button text are too small compared to the design.
The spacing between the top and bottom separators should match


Yep, but we haven't worked on the font-sizes yet. I'm waiting to the font sizes implementation before tunning this on the templates. We haven't worked on setting the spacing either. The objective of this PR is just to add the basic elements of the post-list layout.

File: club/patterns/post-list.php (hunk 1 above)
Comment (matiasbenedetto) [reply club/patterns/post-list.php]: Yep, ready

**Review by jffng — APPROVED** [review]
All the correct elements are there, I just left one question. I'm okay with merging this and iterating on the design once the rest of the styles are in place.

**Pull request merged**